apis/checkpoint/finalized_state.yaml

```diff
@@ -0,0 +1,44 @@
+get:
+  operationId: "getFinalizedCheckpointState"
+  summary: "Get full BeaconState object for finalized checkpoint state"
+  description: |
+     Returns full BeaconState object for a finalized checkpoint state from the WS period.
+     Depending on `Accept` header it can be returned either as json or as bytes serialized by SSZ
```

Comment:
I don't see a reason for this API to support returning JSON. It makes the request significantly more expensive to serve and use more bandwidth, and any client needs to be able to support an SSZ BeaconState.  The debug endpoint is available for humans or other tools that want a more readable version of the spec.